### PR TITLE
Update manage-alerts-windows-defender-advanced-threat-protection.md

### PR DESCRIPTION
**Pull request by joinimran**
The user has suggested adding the attributes of the rule that are supported while the creation of the new rules in the same document. I have added the required information.

Problem Details: https://github.com/MicrosoftDocs/windows-itpro-docs/issues/2974

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/security/threat-protection/windows-defender-atp/manage-alerts-windows-defender-advanced-threat-protection.md [hunk 1]
```diff
@@ -69,6 +69,14 @@ Create custom rules to control when alerts are suppressed, or resolved. You can
 
 2.  Select **Create a supression rule**.
 
+    You can create supression rule based on the following attributes.
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/windows-defender-atp/manage-alerts-windows-defender-advanced-threat-protection.md]: You can create suppression rule based on the following attributes:

**Comment by JohanFreelancer9** [comment]
Joyce please have a look to see what I have done for this PR. Will this
process work?

File: windows/security/threat-protection/windows-defender-atp/manage-alerts-windows-defender-advanced-threat-protection.md (hunk 1 above)
Comment (mypil) [reply windows/security/threat-protection/windows-defender-atp/manage-alerts-windows-defender-advanced-threat-protection.md]: @joinimran - please merge the commit on the copy/edit review made by JohanFreelancer9 or make the necessary changes on the PR following the suggestions he made. Thank you.

**Comment by mypil** [comment]
@joinimran - please check again on the copy/edit that I did and merge the new commit if you can. Thank you.

**Comment by mypil** [comment]
@mjcaparas - can you please assist to check, approve and merge the content for this PR based on issue #2974?

Thank you.

**Pull request merged**